ui/imports/shared/panels/ProfileBioSocialsPanel.qml

```diff
@@ -69,24 +69,18 @@ Control {
 
         StatusScrollView {
             id: scrollView
+
             visible: root.bio
             padding: 0
-
-            Binding on rightPadding {
-                delayed: true
-                value: scrollBar.visible ? 16 : 0
-            }
+            rightPadding: ScrollBar.vertical.visible ? 16 : 0
 
             Layout.maximumHeight: 108
             Layout.fillWidth: true
-            contentHeight: bioText.height
-            implicitHeight: contentHeight
-            ScrollBar.vertical: StatusScrollBar {
-                id: scrollBar
 
-                policy: bioText.height > scrollView.availableHeight
-                        ? ScrollBar.AlwaysOn : ScrollBar.AlwaysOff
-            }
+            contentWidth: availableWidth
+
+            ScrollBar.vertical.policy: bioText.height > scrollView.availableHeight
```

Comment:
Yes I also tried to replace this already, but it won't work that simple. It will auto-hide because of this line:
 https://github.com/status-im/status-desktop/blob/14f328d24d10366ad878e36a4de4eabd6a71102c/ui/StatusQ/src/StatusQ/Controls/StatusScrollBar.qml#L50

And I believe the idea here was to make it always visible, when there's hidden content.